src/features/query-bar/constants.js

```diff
@@ -9,6 +9,9 @@ export const OPERATOR_KEY: 'operatorKey' = 'operatorKey';
 export const VALUE: 'value' = 'value';
 export const VALUE_DISPLAY_TEXT: 'valueDisplayText' = 'valueDisplayText';
 export const VALUE_KEY: 'valueKey' = 'valueKey';
+export const WHERE: null = null;
```

Comment:
why is this null?

Reply:
This can be made into the 'WHERE' string. Initially I had numbers to denote AND and OR but I changed them over to strings so WHERE can be made a string as well.